docs/klaytn-contracts/dao-contracts/introduction.md

```diff
@@ -3,3 +3,54 @@ title: ✨ DAO Contracts
 sidebar_label: Introduction
 ---
 
+## Introduction <a id="DAO Introduction"></a>
+The advent of blockchain technology has resulted in a significant shift in traditional organizational structure. The traditional organizational structure encourages top-level authority to decide on management, operations, and actions, giving coordination to a central authority. Because of the blockchain's decentralized nature, coordination and management of an organization can be delegated to all members, removing the need for centralized control.
+
+In contrast to traditional organizations, DAOs are community-driven entities that do not require central authority or coordination. All members now have management and coordination power, and they can make proposals, cast votes, and make decisions in the best interests of the organization.
+
+The following are some of the benefits of running an organization in a decentralized way:
+
+* **Transparency**: All proposals and votes can be viewed publicly on the blockchain. This enhances transparency and accountability because all members will want to act honestly.
+* **Community-driven**: People with similar interests are brought together to work toward a common goal. As a result, a DAO allows people from all over the world to collaborate on a common goal.
+* **No-central Authority**: Decisions in a DAO are not made by a central authority.
+
+## How does a DAO work? <a id="How do DAO’s work"></a>
+
+Smart contracts can be used to program the functionality of a DAO. Governance abilities such as raising proposals, casting votes, and executing proposals are all possible and readily available on-chain. Remember that all of this is done by members of the community, not by a central authority.
+
+Members, ideally, agree on decisions after raising a proposal in the first instance. If the proposal is supported by the majority of stakeholders, it is automatically implemented. For example, based on the result of a members’ vote, a code may be triggered to mint a predetermined amount of tokens held in the DAO’s custody to a specified blockchain address.  
+
+DAO members frequently invest their funds in the form of tokens to achieve a common goal. These members' tokens typically represent their proportionate voting rights. For example in some DAO's, member who owns 100 tokens has twice the voting power of a member who owns 50 tokens. This practice is viewed positively because individuals who have a greater financial stake in the DAO are more motivated to do what is best for the organization.
+
+Having said that, DAO members have the option of making proposals for the project's future. The following categories represent some of the critical decisions made by DAO members:
```

Comment:
The below 2 points does not look like critical decisions. Can we add like examples from our proposals like https://govforum.klaytn.foundation/t/kgp-6-proposal-to-establish-a-sustainable-and-verifiable-klay-token-economy/157. 

* **Technical proposals**:  These are proposals that can request funding from the DAO's treasury,  make changes to the treasury for the project’s future.
* **Treasury proposals**: These proposals suggest technical improvements or changes to the to the network or consensus to improve the security, performance, or functionality of the protocol.
* **Community proposals**: These proposals that can request funding to engage and grow the community.

Reply:
yeah right...well noted